run/test/specs/utils/capabilities_android.ts

```diff
@@ -44,6 +44,18 @@ const sharedCapabilities: AppiumAndroidCapabilities & AppiumCapabilities = {
 //   "appium:eventTimings": false,
 // };
 
+// const sharedCapabilities: AppiumCapabilities = {
+//   "appium:app": androidAppFullPath,
+//   "appium:platformName": "Android",
+//   "appium:platformVersion": "14",
+//   "appium:appPackage": "network.loki.messenger",
+//   "appium:appWaitActivity":
+//     "org.thoughtcrime.securesms.onboarding.LandingActivity",
+//   "appium:automationName": "UiAutomator2",
+//   "appium:newCommandTimeout": 300000,
+//   "appium:eventTimings": false,
+// };
+
```

Comment:
yeah, it was just because the appWaitActivity got moved and now we have a new capability, so it was just in case it was still there on an old build. But can remove